tests/fixtures/s3.py

```diff
@@ -36,14 +38,9 @@ def __init__(self, image: str, settings: Settings) -> None:
         self.ports = {'9000/tcp': 9100, '9101/tcp': 9101}
         # It's necessary to have 4 volumes to enable versioning (run MinIO in Erasure Code)
         # https://docs.min.io/docs/minio-erasure-code-quickstart-guide.html
-        self.volumes = ['test-data-1:/data1', 'test-data-2:/data2', 'test-data-3:/data3', 'test-data-4:/data4']
+        self.volumes = [f'test-data-{random.randint(100,999)}:/data{i}' for i in range(1, 5)]
```

Comment:
I would suggest to define one random suffix and tag all the volumes with it, so that you know which ones belong to the single container start:
```suggestion
        volume_suffix = os.urandom(5).hex()
        self.volumes = {f'test-data-{volume_suffix}-{i}': {'bind': f'/data{i}'} for i in range(1, 5)}
```